components/semotus/index.js

```diff
@@ -1247,23 +1247,11 @@
 				objectTemplate._queueRemoteCall(this.__id__, propertyName, deferred, arguments);
 
 				if (self.controller && self.controller.handleRemoteError) {
-					deferred.promise.originalThen = deferred.promise.then;
```

Comment:
was this really just entirely dead code that can be removed with no consequence haha

Reply:
This is a good point, we should put this code through a security review before we submit this into the apps.